### PR TITLE
fix: Drop "(Summary)" from slug/title

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: files/en-us/_redirects.txt [hunk 1]
```diff
@@ -5391,7 +5391,8 @@
 /en-US/docs/Games/Anatomy_of_a_vIdeo_game	/en-US/docs/Games/Anatomy
 /en-US/docs/Games/Doc_Status	/en-US/docs/MDN/Doc_status/Games
 /en-US/docs/Games/Introduction/Introduction_3D	/en-US/docs/Games/Techniques/3D_on_the_web
-/en-US/docs/Games/Introduction_to_HTML5_Game_Gevelopment_(summary)	/en-US/docs/Games/Introduction_to_HTML5_Game_Development_(summary)
+/en-US/docs/Games/Introduction_to_HTML5_Game_Development_(summary)	/en-US/docs/Games/Introduction_to_HTML5_Game_Development
+/en-US/docs/Games/Introduction_to_HTML5_Game_Gevelopment_(summary)	/en-US/docs/Games/Introduction_to_HTML5_Game_Development
```
Comment (chrisdavidmills) [inline files/en-us/_redirects.txt:5395]: Line shouldn't be duplicated, right?

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
One comment to address.

File: files/en-us/_redirects.txt (hunk 1 above)
Comment (nschonni) [reply files/en-us/_redirects.txt:5395]: There was a subtle typo "Gevelopment", which is why there was an old redirect

Comment (chrisdavidmills) [reply files/en-us/_redirects.txt:5395]: Ah, thanks, I totally didn't spot that the first time round!

**Review by chrisdavidmills — APPROVED** [review]

**Pull request merged**